contracts/veLON.sol

```diff
@@ -242,6 +242,10 @@ contract veLON is IveLON, ERC721, Ownable, ReentrancyGuard {
         }
 
         emit Deposit(msg.sender, _tokenId, _value, _lockedBalance.end, _depositType, block.timestamp);
+        if (_depositType != DepositType.MERGE_TYPE && _depositType != DepositType.INCREASE_UNLOCK_TIME) {
```

Comment:
Line 240 has checked the `_value != 0 && _depositType != DepositType.MERGE_TYPE`. Maybe we could merge these two `if-clause`.

Reply:
I originally thought to put the emit in line 240 is a better way, however the test just couldn't pass. I'm still figure out what was going on.